### DISCUSSION TITLE
Does Semaphore support PostgreSQL 15

### DISCUSSION THREAD

**Discussion by invisibleninja06** [semaphoreui/semaphore#1834] [Q&A] [answered]
Not finding anywhere the exact supported versions but seen 14 was used in the docker compose example so wondering if semaphore will support PostgreSQL 15 without issue?

Thanks in advance.

**Comment by tboerger** [comment, marked as answer]
Semaphore doesn't use any special postgresql configurations, so this should work without a problem.

**Reply by invisibleninja06** [reply]
Thanks, just wanted to be sure there wasn't any known issues i was missing.
